psamm/commands/sbmlexport.py

```diff
@@ -24,6 +24,15 @@
 class SBMLExport(MetabolicMixin, Command):
     """Export model as SBML file."""
 
+    @classmethod
+    def init_parser(cls, parser):
+        parser.add_argument(
+            'file', type=str, help='File path for writing the Excel workbook',
+            default='-')
```

Comment:
Ooooh, I was looking at `excelexport` for reference and it doesn't use `nargs` or its `init_parser` for some reason, I thought that was the way to go.